src/renderer/store/modules/settings.js

```diff
@@ -174,6 +174,7 @@ const state = {
   defaultQuality: '720',
   defaultTheatreMode: false,
   defaultVideoFormat: 'dash',
+  disableExternalLinkOpening: true,
```

Comment:
There's no reason to disable it by default.
```suggestion
  disableExternalLinkOpening: false,
```